elementor/elementor
# Internal: Conversion Center - Finalize Link In bio Styling

## PR Checklist


## PR Type
What kind of change does this PR introduce?


## Summary

This PR can be summarized in the following changelog entry:

* Finalize Link In Bio Widget

## Description
An explanation of what is done in this PR

* All controls for styiling and adding content are added and wired

## Test instructions
This PR can be tested by following these steps:

* Add The Link In Bio Widget to the editor
* Test all the controls and verify that changes in the controls are applied to the editor preview

## Quality assurance


Fixes #
